examples/raytrace-parallel/index.js

```diff
@@ -0,0 +1,111 @@
+const button = document.getElementById('render');
+const canvas = document.getElementById('canvas');
+const scene = document.getElementById('scene');
+const concurrency = document.getElementById('concurrency');
+const concurrencyAmt = document.getElementById('concurrency-amt');
+const timing = document.getElementById('timing');
+const timingVal = document.getElementById('timing-val');
+const ctx = canvas.getContext('2d');
+
+button.disabled = true;
+concurrency.disabled = true;
+
+// First up, load our wasm
+wasm_bindgen('./raytrace_parallel_bg.wasm')
+  .then(run)
+  .catch(e => {
+    if (e instanceof WebAssembly.CompileError) {
+      let msg = 'failed to compile wasm module';
+      msg += `\nerror: ${e.message}`;
+      msg += "\ncurrently this requires nightly Firefox";
+      alert(msg);
```

Comment:
This already came up in the WebAssembly group, we should have a way to feature test. You could have an unrelated compilation error here.

Reply:
It's true! I believe the feature testing method is intended to be done with tiny wasm modules and something like `new WebAssembly.Instance(buffer)` where `buffer` is like 20 bytes long and does precisely just one operation. I was largely just lazy here :)